pkg/cloud/metadata/lingjun.go

```diff
@@ -0,0 +1,48 @@
+package metadata
+
+import (
+	"encoding/json"
+	"os"
+)
+
+var LingjunConfigFile = "/host/etc/eflo_config/lingjun_config"
+
+type LingjunMetaData struct {
+	RegionId          string   `json:"RegionId"`
+	ZoneId            string   `json:"ZoneId"`
+	NodeId            string   `json:"NodeId"`
+	InstanceType      string   `json:"InstanceType"`
+	EniMac            string   `json:"EniMac"`
+	AckNicName        string   `json:"AckNicName"`
+	NimitzNetworkMode string   `json:"NimitzNetworkMode"`
+	NimitzInterface   []string `json:"NimitzInterface"`
```

Comment:
How about remove fields that we don't use? This way, we:
* reduce the possibility of being breaked by changes of this file.
* slightly improve the performance as we don't need to deserialize these extra fields.